core/services/form/meta/Attributes.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace EventEspresso\core\services\form\meta;
+
+use EventEspresso\core\services\form\meta\inputs\Text;
+use EventEspresso\core\services\json\JsonDataHandler;
+use EventEspresso\core\services\loaders\LoaderFactory;
+
+/**
+ * Class Attributes
+ * For managing HTML attributes for a form element
+ *
+ * @author  Brent Christensen
+ * @package EventEspresso\core\services\form\meta
+ * @since   $VID:$
+ */
+class Attributes
+{
+
+    /**
+     * @var JsonDataHandler
+     */
+    private $json_data_handler;
+
+    /**
+     * @var InputTypes
+     */
+    private $input_types;
+
+    /**
+     * @var array
+     */
+    private $attributes = [];
+
+    /**
+     * @var array
+     */
+    private $attribute_types = [
+        'accept'          => 'string',
+        'accesskey'       => 'string',
+        'alt'             => 'string',
```

Comment:
i just copied all of these in here for now, but we can decide which to keep if we want to restrict things. I didn't add any of the fields that pertain to scripts, such as onChange, etc

Reply:
Nice